internal/github/github.go

```diff
@@ -0,0 +1,180 @@
+package github
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/google/go-github/github"
+	"golang.org/x/oauth2"
+)
+
+var (
+	client *github.Client
+	ctx    context.Context
+	tc     *http.Client
+)
+
+func init() {
+	ctx = context.Background()
+	ts := oauth2.StaticTokenSource(
+		&oauth2.Token{AccessToken: os.Getenv("GITHUB_ACCESS_TOKEN")},
+	)
+	tc = oauth2.NewClient(ctx, ts)
+	client = github.NewClient(tc)
+}
+
+func GetHTMLURLs(issues []*github.Issue) []string {
+	var urls []string
+	for _, iss := range issues {
+		urls = append(urls, iss.GetHTMLURL())
+	}
+	return urls
+}
+
+func GetOpenIssues(repoURLs []string) []*github.Issue {
+	var all []*github.Issue
+	opts := github.IssueListByRepoOptions{
+		State: "open",
+	}
+	for _, url := range repoURLs {
+		fragments := strings.Split(string(url), "/")
+		owner := fragments[len(fragments)-2]
+		name := strings.TrimSuffix(fragments[len(fragments)-1], ".git")
+		
+		issues, resp, err := client.Issues.ListByRepo(ctx, owner, name, &opts)
```

Comment:
default page size may not enough: https://github.com/google/go-github#pagination

Reply:
Indeed. However, if open issues remain after processing the default batch, you can just rerun the program and it will continue from where it left off.

I'll be happy to implement paging, but I'd prefer to in a separate PR, as this would require research and redoing many things now, when it's probable that all the open issues will be processed in one or two runs.